luaui/Widgets/ana_report_widgets.lua

```diff
@@ -0,0 +1,132 @@
+local widget = widget ---@type Widget
+
+function widget:GetInfo()
+    return {
+        name    = "Analytics - Widgets",
+        desc    = "Report widget usage to Analytics API",
+        author  = "uBdead",
+        date    = "Oct 2025",
+        license = "GPL-v2",
+        layer   = 0,
+        enabled = true,
+        handler = true,
+    }
+end
+
+-- and it will filter out inconsequential matches
+local REPORT_DELAY_FRAMES = 30
+local MAX_WIDGETS_PER_REPORT = 50
+local HASH_PER_FRAME = 1
+local EXCLUDE_ZIP = true
+
+local initialized = false
+local widgets = {}
+local currentWidgetIndex = 1
+
+-- from gui_options.lua by WatchTheFort
+local function GetWidgetToggleValue(widgetname)
+	if widgetHandler.orderList[widgetname] == nil or widgetHandler.orderList[widgetname] == 0 then
+		return false
+	elseif widgetHandler.orderList[widgetname] >= 1
+		and widgetHandler.knownWidgets ~= nil
+		and widgetHandler.knownWidgets[widgetname] ~= nil then
+		if widgetHandler.knownWidgets[widgetname].active then
+			return true
+		else
+			return 0.5
+		end
+	end
+end
+
+local function processWidget(widget)
+    -- I know this is redundant as we literally just defined these values ourselves, 
+    -- but i want to keep the GetWidgetToggleValue the same as the original
+    local state = GetWidgetToggleValue(widget.name)
+    if state == 0 then   
```

Comment:
The function returns false/0.5/true rather than 0/0.5/1 so this shouldn't ever pass